### DISCUSSION TITLE
Extend base cargo commands

### DISCUSSION THREAD

**Discussion by pepperoni505** [sagiegurari/cargo-make#1114] [Q&A] [answered]
Is it possible to extend base cargo commands like `cargo clean` to include additional steps, or will I need to have a custom clean command?

**Comment by sagiegurari** [comment, marked as answer]
you can extend the task in your make file and add depedencies to do things before the clean.
if you want to do things after the clean, you need to do clear=true and than set run action to the tasks you want to run.
there are a lot of examples in the internal makefiles of how i do things.
